vita3k/gui/src/perf_overlay.cpp

```diff
@@ -68,19 +70,19 @@ void draw_perf_overlay(GuiState &gui, EmuEnvState &emuenv) {
     ImGui::PushStyleVar(ImGuiStyleVar_ChildRounding, 5.f * emuenv.dpi_scale);
     ImGui::BeginChild("#perf_stats", WINDOW_SIZE, true, ImGuiWindowFlags_NoMove | ImGuiWindowFlags_NoDecoration | ImGuiWindowFlags_NoSavedSettings);
     if (emuenv.cfg.performance_overlay_detail == PerfomanceOverleyDetail::MINIMUM)
-        ImGui::Text("FPS: %d", emuenv.fps);
+        ImGui::Text("Avg: %u (%u ms)", emuenv.avg_fps, (uint16_t)(1000 / (emuenv.avg_fps ? emuenv.avg_fps : 1)));
```

Comment:
revert here, minimum is just display normal fps, like if you using full screen

Reply:
its average because now fps changes constantly it is literally unreadable, and because avg is on the last 60 frames it is a lot more stable and readable and because 60 frames is a reasonable ammount of time its better to use that to give an idea on how good the game is running